worlds/alttp/ItemPool.py

```diff
@@ -371,14 +372,30 @@ def generate_itempool(world):
 
     dungeon_items = [item for item in get_dungeon_item_pool_player(world, player)
                      if item.name not in world.worlds[player].dungeon_local_item_names]
-
+    dungeon_item_replacements = difficulties[world.difficulty[player]].extras[0]\
+                                + difficulties[world.difficulty[player]].extras[1]\
+                                + difficulties[world.difficulty[player]].extras[2]\
+                                + difficulties[world.difficulty[player]].extras[3]\
+                                + difficulties[world.difficulty[player]].extras[4]
+    world.random.shuffle(dungeon_item_replacements)
     if world.goal[player] == 'icerodhunt':
         for item in dungeon_items:
             world.itempool.append(ItemFactory(GetBeemizerItem(world, player, 'Nothing'), player))
             world.push_precollected(item)
     else:
+        for x in range(len(dungeon_items)-1, -1, -1):
+            item = dungeon_items[x]
+            logging.info(f'itemn:{x} item:{item} len:{len(dungeon_items)}')
```

Comment:
these should be .debug level logs at most.

Reply:
Whoops, this wasn't supposed to be left in there